### PR TITLE
Mac: Create the `bundle` target and modify affected shell scripts

### LINKED ISSUE #2259: Mac: Custom `make` target of `bundle`
So almost have a PR ready to upload to GH but have a few questions first:

1. Should the PR be submitted to `master` branch or `v2.1` branch? :question: *(Still unclear when I should be doing one or the other... bug fixes to v2.1 but this is more a RFE/RFC)*
2. Verification for Travis CI from #2245 would be appreciated e.g. Did I miss anything any notes on changes to .travis.yml from that issue? :question:

Ref:
* https://github.com/QupZilla/qupzilla/pull/2226#discussion_r101901152

### PR DESCRIPTION
**Pull request by Martii**
* Very minor bug fix with `bless` switch. Syntax says requires double hyphens but does currently work with single. Preventative measure.
* Indentation fixes... didn't notice original was 1 spaced indent instead of my usual 2 space indent
* Documented why this missing lib(s) are missing... if the QupZilla source is changed to accommodate this down the line it should do it dynamically without the correction
* Change BUILDING.md and README.md accordingly.

---

Closes #2259

### CODE REVIEW

**Comment by Martii** [comment]
Still needs Travis CI confirmation of any changes needed from https://github.com/QupZilla/qupzilla/pull/2245#discussion_r106724757 Cc: @srazi

**Comment by srazi** [comment]
@Martii Yep. Please add necessary changes to your PR. But that part of travis.yml script runs on new tag so maybe you want to test it on your fork by pushing a new tag (you have to update api_token for your fork).

**Comment by Martii** [comment]
@srazi 
> Yep.

Thanks for the verification. With the Mac it's easy to break things and I'm in uncharted waters with Travis CI. :)

> so maybe you want to test it on your fork by pushing a new tag (you have to update api_token for your fork).

Trying to avoid that for now and let GH do its thing... I have too many accounts all over the place right now and heading off for the weekend in a bit. I still don't have Travis installed locally on the Mac *(or even Linux for that matter)* thus taking a bit longer. Will delve into that on my lengthy TODO list though. :)

This is basically for coding/building with deploy symmetry as it appears logically that it could work without that change to `.travis.yml`.

File: .travis.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ script:
 before_deploy:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
          cd $QZ_DIR;
-         ./mac/macdeploy.sh $MACDEPLOYQT;
+         make bundle
```
Comment (srazi) [inline .travis.yml]: Missing `;`

Comment (Martii) [inline .travis.yml]: Thanks... nearing brain dead for the week. ;) :smile:

**Comment by Martii** [comment]
Hmmm.... getting a permission denied when I check this PR out and merge it into `master` HEAD...

**Comment by Martii** [comment]
Okay... retested via merge of PR to local branch and installs/exec's from the human interaction side. Should be PR READY unless any more suggestions.

Misc rant:
Really need to get a better VNC Client/Server for the Mac as the default screen sharing Apple gives is still pretty bad even with the headless dongle I got for it. Clipboard actions aren't shared with the Host which super bites and it's still a bit laggy compared to my other VNC'd systems... more TODO.

**Pull request merged**